src/Stats/DateSlider.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import PropTypes from 'prop-types';
+import { Button } from 'react-native-elements';
+
+export default function SubHeader(props) {
```

Comment:
File name and component should match or atleast close to each other

Reply:
Fixed, replace `SubHeader` by `DateSlider`